v1.0/openapi3/schemas/AmountType.yaml

```diff
@@ -3,5 +3,10 @@ type: string
 enum:
   - SEND
   - RECEIVE
-description: "Below are the allowed values for the enumeration AmountType.\n- SEND - Amount the Payer would like to send, that is, the amount that should be withdrawn from the Payer account including any fees.\n\n- RECEIVE - Amount the Payer would like the Payee to receive, that is, the amount that should be sent to the receiver exclusive of any fees."
+description: |
+  Below are the allowed values for the enumeration AmountType.\n
+  - SEND - Amount the Payer would like to send, that is, the amount that should
+  be withdrawn from the Payer account including any fees.\n\n
```

Comment:
Just curious, why do we need two newline chars to separate the desc for two different enum types?

Reply:
No idea. That's how it was in the original spec file. We can address formatting later.

Reply:
Okay 👍